Pully Penelope/Assets/Scripts/PlayerMovement.cs

```diff
@@ -24,12 +24,16 @@ public class PlayerMovement : MonoBehaviour
     private bool isGrabbingHandle;
     private bool shouldUpdateAnimation = true;
 
+    private AudioSource attachSound;
+    private bool hasPlayedAttachSound = false;
+
     private void Start()
     {
         rigidbody = GetComponent<Rigidbody2D>();
         animator = GetComponent<Animator>();
         animator.SetFloat("orientationX", 0);
         animator.SetFloat("orientationY", -1);
+        attachSound = GameObject.Find("AttachSound").GetComponent<AudioSource>();
```

Comment:
serialize instead?

Reply:
So this is the same thing where since it's on the player I used GetComponent().